modules/bag/tests/tests.cpp

```diff
@@ -35,8 +35,8 @@ constexpr auto ROBOT_MSG_PERIOD = std::chrono::milliseconds{ 1 };
 constexpr std::size_t FLEET_MSG_COUNT = 5;
 constexpr auto FLEET_MSG_PERIOD = std::chrono::milliseconds{ 2 };
 constexpr auto SENDER_ID = "bag_tester";
-constexpr auto ROBOT_TOPIC = "robot";
-constexpr auto FLEET_TOPIC = "fleet";
+constexpr auto ROBOT_TOPIC = "bag_test/robot";
+constexpr auto FLEET_TOPIC = "bag_test/fleet";
```

Comment:
adding a prefix to make sure that if a topic is publishing it doesn't make the test fail